packages/taiko-client/bindings/encoding/protocol_config.go

```diff
@@ -0,0 +1,55 @@
+package encoding
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/params"
+
+	"github.com/taikoxyz/taiko-mono/packages/taiko-client/bindings"
+)
+
+var (
+	livenessBond, _             = new(big.Int).SetString("125000000000000000000", 10)
+	InternlDevnetProtocolConfig = &bindings.TaikoDataConfig{
+		ChainId:               params.TaikoInternalL2ANetworkID.Uint64(),
+		BlockMaxProposals:     324_000,
+		BlockRingBufferSize:   324_512,
+		MaxBlocksToVerify:     16,
+		BlockMaxGasLimit:      240_000_000,
+		LivenessBond:          livenessBond,
+		StateRootSyncInternal: 16,
+		OntakeForkHeight:      374_400,
+	}
+	HeklaProtocolConfig = &bindings.TaikoDataConfig{
+		ChainId:               params.HeklaNetworkID.Uint64(),
+		BlockMaxProposals:     324_000,
+		BlockRingBufferSize:   324_512,
+		MaxBlocksToVerify:     16,
+		BlockMaxGasLimit:      240_000_000,
+		LivenessBond:          livenessBond,
+		StateRootSyncInternal: 16,
+		OntakeForkHeight:      540_000,
+	}
+	MainnetProtocolConfig = &bindings.TaikoDataConfig{
+		ChainId:               params.TaikoMainnetNetworkID.Uint64(),
+		BlockMaxProposals:     324_000,
+		BlockRingBufferSize:   324_512,
```

Comment:
```suggestion
		BlockRingBufferSize:   360_000,
```

Reply:
`InternlDevnet` and `Hekla` are also `360_000`?

Reply:
No, Hekla is `324_512`: https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/contracts/hekla/HeklaTaikoL1.sol#L17